packages/react/src/date-picker/DatePicker.test.tsx

```diff
@@ -68,18 +67,6 @@ function ControlledDatePicker({
 	);
 }
 
-function ClearableDatePicker({ initialValue }: { initialValue?: Date }) {
-	const [value, setValue] = useState<Date | undefined>(initialValue);
-	return (
-		<Stack gap={4} alignItems="flex-start">
-			<DatePicker label="Clearable" value={value} onChange={setValue} />
-			<Button data-testid="clear" onClick={() => setValue(undefined)}>
-				Clear
-			</Button>
-		</Stack>
-	);
-}
```

Comment:
This test was flaky after the updates. It also doesn't provide a lot of value above existing tests since it's updating external state in a different way to other tests but not updating the component in a different way, i.e. both receive updated values from a `useState` hook. So, I removed it.